### PR TITLE
:tada: first version of the script

### PR DESCRIPTION
**Pull request by yodur2potassium**
This PR intends to merge the first version of the script for test. Next steps are env variables for token and logfile.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.txt
```
Comment (louckousse) [inline .gitignore]: Why not use an extension of our own?

File: init.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+echo "Current date and time is : ${CURRENT_DATE}"
+echo "Don't order before: ${NOT_BEFORE}"
+
+[[ -f next-orders.txt ]] && echo "File for next orders found" || touch next-orders.txt
+[[ -f sent-orders.txt ]] && echo "File for sent orders found" || touch sent-orders.txt
+
+if [[ `tail -1 next-orders.txt` < $CURRENT_DATE ]];then
+	curl -H "Authorization: Bearer changeit" \
+		-X POST \
+		"https://hooks.zapier.com/hooks/catch/3005275/o28kvwa/" \
```
Comment (louckousse) [inline init.sh]: We have to remove this link and put it in an env file.

**Review by louckousse — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (yodur2potassium) [reply .gitignore]: Clever!

File: init.sh (hunk 2 above)
Comment (yodur2potassium) [reply init.sh]: Done.

**Review by louckousse — APPROVED** [review]

**Pull request merged**